lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -561,6 +561,60 @@ ospf:
     config_get_token: ['/^router ospf <name>$/i', '/^vrf\s+(\S+)$/']
     config_set: ["router ospf <name>", "<state> vrf <vrf>", "end"]
 
+radius_server:
+  hosts:
+    config_get: "show running-config radius"
+    config_get_token: '/^radius-server host ((?:[0-9]{1,3}\.){3}[0-9]{1,3})/'
+    config_set: '<state> radius-server host <ip>'
+    default_value: ''
+
+  auth-port:
+    config_get: "show running-config radius"
+    config_get_token: '/^radius-server host %s.* auth-port (\d+).*$/'
+    config_set: '<state> radius-server host <ip> auth-port <port>'
+    default_value: ''
```

Comment:
Based on what I see on the device it looks like the following are the default values:

dt-n9k5-1(config)# sh radius-server 
timeout value:5
retransmission count:1
deadtime value:0
source interface:any available
total number of servers:3

following RADIUS servers are configured:
        5.5.5.5:
                available for authentication on port: 1812
                available for accounting on port: 1813
                timeout:9
        9.9.9.9:
                available for authentication on port: 1812
                available for accounting on port: 1813
        55::5:
                available for authentication on port: 1812
                available for accounting on port: 1813
dt-n9k5-1(config)# 

1) timeout value: 5
2) retransmission value: 0 
3) accounting port: 1813
4) authentication port: 1812

Also based the the output accounting and authentication are enabled by default so they should be 'true' not 'false'